internal/commands/completion.go

```diff
@@ -31,13 +41,26 @@ To configure your bash shell to load completions for each session, add the follo
 			}
 			completionPath := filepath.Join(packHome, "completion")
 
-			if err := cmd.Parent().GenBashCompletionFile(completionPath); err != nil {
+			var flagErr error
+			switch flags.Shell {
+			case "bash":
+				flagErr = cmd.Parent().GenBashCompletionFile(completionPath)
+			case "zsh":
+				flagErr = cmd.Parent().GenZshCompletionFile(completionPath)
+			default:
+				return errors.Errorf("%s is unsupported shell", flags.Shell)
+			}
+
+			if flagErr != nil {
 				return err
 			}
 
+			logger.Infof("Completion File for %s is created", flags.Shell)
```

Comment:
In order for `. $(pack completion --shell zsh)` to work, nothing else can be printed out. This message should be removed.